chain/jsonrpc-primitives/src/types/light_client.rs

```diff
@@ -1,25 +1,31 @@
+#[cfg(feature = "schemars")]
+use schemars;
```

Comment:
No need to import a crate since we use the fully qualified names to JsonSchema anyway